mvc/src/main/java/nextstep/mvc/DispatcherServlet.java

```diff
@@ -33,36 +35,48 @@ public void addHandlerMapping(final HandlerMapping handlerMapping) {
         handlerMappings.add(handlerMapping);
     }
 
+    public void addHandlerAdapter(final HandlerAdapter handlerAdapter) {
+        handlerAdapters.add(handlerAdapter);
+    }
+
     @Override
-    protected void service(final HttpServletRequest request, final HttpServletResponse response) throws ServletException {
+    protected void service(final HttpServletRequest request, final HttpServletResponse response)
+            throws ServletException {
         log.debug("Method : {}, Request URI : {}", request.getMethod(), request.getRequestURI());
 
         try {
-            final var controller = getController(request);
-            final var viewName = controller.execute(request, response);
-            move(viewName, request, response);
-        } catch (Throwable e) {
+            final var handler = getHandler(request);
+            if (Objects.isNull(handler)) {
+                noHandlerFound(request, response);
+                return;
+            }
+
+            final HandlerAdapter handlerAdapter = getHandlerAdapter(handler);
+            final ModelAndView mv = handlerAdapter.handle(request, response, handler);
```

Comment:
```suggestion
            final ModelAndView modelAndView = handlerAdapter.handle(request, response, handler);
```
a : 약어를 사용하는 것보다는 길더라도 구체적인 이름을 명시해주는게 좋더라구요 🧚‍♀️